src/taco-cli/cli/kit.ts

```diff
@@ -510,13 +520,15 @@ class Kit extends commands.TacoCommandBase {
      * Pretty prints the platform and plugin update information
      */
     private static printProjectUpdateInfo(id: string, installedPlatformVersions: IDictionary<string>, installedPluginVersions: IDictionary<string>,
-        platformVersionUpdates: IDictionary<string> = null, pluginVersionUpdates: IDictionary<string> = null): void {
+        platformVersionUpdates: IDictionary<string> = null, pluginVersionUpdates: IDictionary<string> = null,
+        pluginsInConfigVersions: IDictionary<string> = null, pluginsInConfigUpdates: IDictionary<string> = null): void {
         var indent: number = LoggerHelper.getDescriptionColumnIndent(Kit.getLongestPlatformPluginLength(installedPlatformVersions ? Object.keys(installedPlatformVersions) : null, installedPluginVersions ? Object.keys(installedPluginVersions) : null));
 
         var platformsRequireUpdate: boolean = Kit.projectComponentNeedsUpdate(installedPlatformVersions, platformVersionUpdates);
         var pluginsRequireUpdate: boolean = Kit.projectComponentNeedsUpdate(installedPluginVersions, pluginVersionUpdates);
+        var removesPluginsFromConfig: boolean = Kit.projectComponentNeedsUpdate(pluginsInConfigVersions, pluginsInConfigUpdates);
```

Comment:
It's another option.
Personally removesPluginsFromConfig makes me thing of a boolean
And removedPluginsFromConfig makes me thing of an array of plugins.
What do those names tell to you? If more people find the removed more intuitive, then I'll change it